test/system/700-play.bats

```diff
@@ -625,6 +625,9 @@ spec:
 
     run_podman kube play --wait $fname
 
+    # debug to see what container is being left behind after the cleanup
+    run_podman ps -a
+    run_podman pod ps
     # there should be no containers running or created
     run_podman ps -aq
```

Comment:
Why not just make this one `ps -a --noheading`? I suggested that way back in #17821 but maybe it got lost? Or maybe there's a reason that's not a good idea? Anyhow, I suspect that the delay added by `ps + pod ps`, above, will be enough to reduce the race condition even further. (Making it harder to track down, that is, instead of fixing the bug).

Reply:
Apologies, I missed that. Fixed to use --noheading. Thanks!